examples/httpapi/requests_storm_example.py

```diff
@@ -39,7 +40,7 @@ def main(argv):
 
     resp = sess.get(url, json=data, stream=True)
     for chunk in resp.iter_content(chunk_size=None, decode_unicode=True):
-        mesg = json.loads(chunk)
+        mesg = s_json.loads(chunk)
```

Comment:
Are we able to us stdlib `json` in the HTTPAPI examples? That way the examples are more standalone and don't rely on any implicit `synapse` functionality?